lib/routes/components.js

```diff
@@ -85,6 +85,12 @@ module.exports = app => {
 			if (component.type === 'module' && component.resources.demos) {
 				demos = await app.repoData.listDemos(component.repo, component.id);
 
+				demos.forEach(async demo => {
+					if (demo.display.html) {
+						demo.display.plainHTML = await fetchHTML(demo.supportingUrls.html);
```

Comment:
I'm also pretty tempted to ignore errors in loading the plain HTML, as then we mitigate against the build service having issues. Then if the plainHTML isn't set, we fall back to iframes again? This might be overcomplicating, can't decide, but we're now relying on multiple back-end systems and it might be nice to ensure the UI doesn't break :)